client/gutenboarding/store/reducer.ts

```diff
@@ -25,14 +24,26 @@ const siteTitle: Reducer< string, ReturnType< typeof Actions[ 'setSiteTitle' ] >
 	state = '',
 	action
 ) => {
-	switch ( action.type ) {
-		case ActionType.SET_SITE_TITLE:
-			return action.siteTitle;
+	if ( action.type === ActionType.SET_SITE_TITLE ) {
+		return action.siteTitle;
 	}
 	return state;
 };
 
-const reducer = combineReducers( { siteType, siteTitle } );
+const verticalSearches: Reducer<
+	Record< string, Vertical[] >,
```

Comment:
Would it make sense to define this as `VerticalSearchType` somewhere, to make it a bit more self-explanatory?

Reply:
It doesn't seem like pulling this part of the type out into a type alias adds a lot of clarity. If you prefer that, we can change it.